customizer/samples/microsoft-teams/Share to Teams for CNX 7.0/README.md

```diff
@@ -1,4 +1,4 @@
-# Share to Microsoft Teams
+# Share to Microsoft Teams 7.0
```

Comment:
Minor issue - could you change this to "Enabling Share to Microsoft Teams for Connections 7.0" ?

Reply:
README updated